cli/script/command-parser.ts

```diff
@@ -599,6 +609,15 @@ function createCommand(): cli.ICommand {
                 }
                 break;
 
+            
+            case "debug":
+                cmd = { type: cli.CommandType.debug };
+
+                var debugCommand = <cli.IDebugCommand>cmd;
```

Comment:
nit: This name clashes with the import/require you had above. Could we rename one of them to something else?